src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/AddressInformationForJson.java

```diff
@@ -0,0 +1,13 @@
+package de.internetsicherheit.brl.bloxberg.cache.ethereum;
+
+public class AddressInformationForJson {
```

Comment:
This a good example of a plain data class. Nothing to change here just wanted to highlight this.
However, I am wondering, what is the difference to `BlockTransaction `?

Reply:
This was @moekappels  code for parsing the content that should be included in the JSON file. I have just commited changes to remove that class and, effectively, use BlockTransaction.

Reply:
i want the PR be made against my last commit c7bdca9a8b1ce4891cfaa95edaa6468bd1bb2a51
How can we solve this?

Reply:
PR is always made between HEAD of the branch and HEAD of master. Pushing to the branch corresponding to the PR automatically updates it.

Reply:
I answer back here. After seeing @moekappels changes about the timestamp, AddressInformationForJson (which now I call it InformationForJson) is necessary because it does not include only data of the transaction.